ui-detail/src/main/java/com/sfaxdroid/detail/utils/DetailUtils.kt

```diff
@@ -26,7 +26,7 @@ import com.sfaxdroid.detail.ActionTypeEnum
 import java.io.File
 import java.io.IOException
 
-internal class DetailUtils {
+class DetailUtils {
```

Comment:
internal class

Reply:
I made the class public so that this class is usable outside of UI detail module. I'll make it internal again when we move compose detail file to ui detail